diff --git a/tools/font/bdfconv/Makefile b/tools/font/bdfconv/Makefile
index 7efea373..d6a0f942 100644
--- a/tools/font/bdfconv/Makefile
+++ b/tools/font/bdfconv/Makefile
@@ -76,6 +76,10 @@ kalpurush: bdfconv
 polish: bdfconv
 	./bdfconv -f 1 -v -m '32-127,$$104-$$107,$$118-$$119,$$141-$$144,$$15a-$$15b,$$179-$$17c,$$d3,$$f3' ../bdf/unifont.bdf -n unifont_polish -o unifont_polish.c -d ../bdf/7x13.bdf
 
+# contains polish characters with the degree symbol
+polish2: bdfconv
+	./bdfconv -f 1 -v -m '32-127,176,211,243,260-263,280-281,321-324,346-347,377-380' ../bdf/unifont.bdf -n unifont_polish2 -o unifont_polish2.c -d ../bdf/7x13.bdf
+
 helvb18: bdfconv
 	./bdfconv -f 1 -m '32-127,x32-64,x91-96,x123-191,x247,x697-879' -p 10 ../bdf/helvB18.bdf -o helvb18_tr.c -n u8g2_font_helvB18_tr -k helvb18_tr_k.c
 
